moped-etl/arcgis/README.md

```diff
@@ -22,6 +22,8 @@ The fourth layer is sourced from a derivative view, `exploded_component_arcgis_o
 
 ## Running the Script
 
+1. (If running full refresh) Ensure the local Moped stack is running with a current snapshot.
```

Comment:
🙏 glad to have some readme improvements in this PR! I just noticed that all of the numbered steps are the number 1 for some reason - this was probably me 🙋. Could you update while you are in here to be 1, 2, 3, 4?

Reply:
TIL! I was wondering what the heck was going on. 😅